index.html

```diff
@@ -145,7 +145,7 @@
           <ul>
             <li class="active withripple" data-target="#about">Material Design for Bootstrap</li>
             <li class="withripple" data-target="#getting-started">Getting Started</li>
-            <li><a href="http://fezvrasta.github.com/bootstrap-material-design/bootstrap-elements.html" target="_blank">Bootstrap elements <i class="mdi-action-open-in-new"></i></a></li>
```

Comment:
This was pointing to a public site, instead of using the local file for testing.  This shouldn't affect gh-pages when published.